src/gluonts/model/deepvar_hierarchical/_estimator.py

```diff
@@ -0,0 +1,308 @@
+        during inference. This is a model optimization that does not affect
+        the accuracy (default: 100)
+    dropout_rate
+        Dropout regularization parameter (default: 0.1)
+    cardinality
+        Number of values of each categorical feature (default: [1])
+    embedding_dimension
+        Dimension of the embeddings for categorical features
+        (default: 5])
+    distr_output
+        Distribution to use to evaluate observations and sample predictions
+        (default: LowrankMultivariateGaussianOutput with dim=target_dim and
+        rank=0, i.e., covariance matrix is assumed to be diagonal).
+        Note that target dim of the DistributionOutput and the estimator constructor
+        call need to match.
+    rank
+        **Not used**. Multivariate Gaussian with diagonal covariance matrix is used.
+        This means setting rank = 0.
+    scaling
+        Whether to automatically scale the target values (default: true)
+    pick_incomplete
+        Whether training examples can be sampled with only a part of
+        past_length time-units
+    lags_seq
+        Indices of the lagged target values to use as inputs of the RNN
+        (default: None, in which case these are automatically determined
+        based on freq)
+    time_features
+        Time features to use as inputs of the RNN (default: None, in which
+        case these are automatically determined based on freq)
+    conditioning_length
+        Set maximum length for conditioning the marginal transformation
+    use_marginal_transformation
+        **Not used**. It is set to False.
+    batch_size
+        The size of the batches to be used training and prediction.
+    """
+
+    @validated()
+    def __init__(
+        self,
+        freq: str,
+        prediction_length: int,
+        target_dim: int,
+        S: np.ndarray,
+        num_samples_for_loss: int = 200,
+        likelihood_weight: float = 0.0,
+        CRPS_weight: float = 1.0,
+        assert_reconciliation: bool = False,
+        coherent_train_samples: bool = True,
+        coherent_pred_samples: bool = True,
+        trainer: Trainer = Trainer(),
+        context_length: Optional[int] = None,
+        num_layers: int = 2,
+        num_cells: int = 40,
+        cell_type: str = "lstm",
+        num_parallel_samples: int = 100,
+        dropout_rate: float = 0.1,
+        cardinality: List[int] = [1],
+        embedding_dimension: int = 5,
+        distr_output: Optional[DistributionOutput] = None,
+        rank: Optional[int] = 0,
+        scaling: bool = True,
+        pick_incomplete: bool = False,
+        lags_seq: Optional[List[int]] = None,
+        time_features: Optional[List[TimeFeature]] = None,
+        conditioning_length: int = 200,
+        use_marginal_transformation: bool = False,
+        batch_size: int = 32,
+        warmstart_epoch_frac: float = 0.0,
+        sample_LH: bool = False,
+        seq_axis: List[int] = None,
```

Comment:
This seems to be missing in docstring? It is not clear what it is either.

Reply:
Yes, updated.